cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -121,26 +121,29 @@ LocalTrajectoryBuilder::AddRangeData(const common::Time time,
 
   if (num_accumulated_ >= options_.scans_per_accumulation()) {
     num_accumulated_ = 0;
-    return AddAccumulatedRangeData(
-        time, sensor::TransformRangeData(accumulated_range_data_,
-                                         tracking_delta.inverse()));
+    // Transforms 'accumulated_range_data_' into a frame where gravity direction
```

Comment:
Since we now have `terminology.rst`: Prefer to make the code self describing? How about renaming `TransformAndFilterRangeData()` to reflect that it is into the gravity aligned frame? `TransformToGravityAlignedFrameAndFilter()`?